extensions/ql-vscode/src/model-editor/shared/model-evaluation-run-state.ts

```diff
@@ -0,0 +1,19 @@
+import { VariantAnalysisStatus } from "../../variant-analysis/shared/variant-analysis";
+import type { VariantAnalysis } from "../../variant-analysis/shared/variant-analysis";
+
+export interface ModelEvaluationRunState {
+  isPreparing: boolean;
+  variantAnalysis: VariantAnalysis | undefined;
+}
+
+export function modelEvaluationRunIsRunning(
+  run: ModelEvaluationRunState,
+): boolean {
+  return (
+    run.isPreparing ||
+    !!(
+      run.variantAnalysis &&
+      run.variantAnalysis.status !== VariantAnalysisStatus.InProgress
```

Comment:
Is this correct? I'd expect that we want 

```suggestion
      run.variantAnalysis.status === VariantAnalysisStatus.InProgress
```